### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by vermasavita**

### CODE REVIEW

File: src/components/Filter/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import './filter.css'
+const Filter = () => {
+    return(
+        <div className="product-listing">
+            <div className="filter">
+                <div className="filter-heading">
+                    Filters
+                    <a href="#">Clear</a>
```
Comment (sakshi006) [inline src/components/Filter/Filter.jsx:8]: why a href? You can use a button or some other element for Clear functionality

**Review by sakshi006 — APPROVED** [review]
Added a comment, rest looks good.

File: src/components/Filter/Filter.jsx (hunk 1 above)
Comment (vermasavita) [reply src/components/Filter/Filter.jsx:8]: Yep.. i think you are right. 

I did not notice this earlier when making the screen in html css. But let me correct this now.

Comment (vermasavita) [reply src/components/Filter/Filter.jsx]: Fixed

**Review by starver20 — APPROVED** [review]
Code looks good to merge. You dont need to make a pr for each page, you can make just one pr for all page conversion to react.

**Comment by vermasavita** [comment]
@starver20 Oh, is it? I did not know that. Thanks for informing.

**Pull request merged**